src/index.js

```diff
@@ -102,59 +109,272 @@ const menu = async () => {
     }
   };
 
+  const battle = async (enemy, player, playerDmg, enemyDmg) => {
+    try {
+      let turn;
+      console.log(`You found ${enemy.name}!\n`);
+      if (player.agi >= enemy.agi) {
+        console.log("You attack first!\n");
+        turn = 1;
+      } else {
+        console.log("The bug attacks first!\n");
+        turn = 0;
+      }
+      while (player.hp > 0 && enemy.hp > 0) {
+        console.log(`${player.name}: ${player.hp}\n${enemy.name}: ${enemy.hp}`);
+        if (turn) {
+          if (playerDmg >= 0) {
+            enemy.hp -= playerDmg;
+            console.log(`You dealt ${playerDmg} dmg to the bug\n`);
+          } else {
+            console.log("Your dmg is null against this bug!\n");
+          }
+          turn--;
```

Comment:
considering turns can only be 0 and 1, why not guarantee that with `turn = 0` and `turn = 1`?